### PR TITLE
KSQL-1802: First draft of KSQL/C3 integration topic

### PR DESCRIPTION
**Pull request by JimGalasyn**
First attempt to capture various details around integrating KSQL with Confluent Control Center.

### CODE REVIEW

File: docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+properties in the KSQL Server configuration file at 
+``<path-to-confluent>/etc/ksql/ksql-server.properties`` and the |c3-short|
+configuration file at 
+``<path-to-confluent>/etc/confluent-control-center/control-center.properties``.
+
+* In the KSQL Server configuration file, set the :ref:`ksql-listeners` property
+  to the IP address of the REST API endpoint for KSQL Server. Typical values
+  are ``http://0.0.0.0:8088`` and ``http://localhost:8088``.
+* In the |c3-short| configuration file, set the ``confluent.controlcenter.ksql.url``
+  property to the KSQL Server hostname. This setting specifies how |c3-short|
```
Comment (big-andy-coates) [inline docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst]: > set the ``confluent.controlcenter.ksql.url`` property to the KSQL Server hostname

I'm assuming this 'url' property would require more than just the hostname? It's probably the full `http://some_host_name:8088`?

File: docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+In the KSQL Server configuration file, set ``listeners`` to bind to all
+interfaces:
+
+::
+
+    listeners=http://0.0.0.0:8088
+
+
+In the |c3-short| configuration file, set ``ksql.url`` to the KSQL Server
```
Comment (big-andy-coates) [inline docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst]: > set ``ksql.url``

Is this the same setting as what you've called ``confluent.controlcenter.ksql.url`` above?

File: docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+hostname, which must be reachable from the host that |c3-short| is installed
+on. Also, set ``ksql.advertised.url`` to the public IP address published by the
```
Comment (big-andy-coates) [inline docs/installation/server-config/integrate-ksql-with-confluent-control-center.rst]: likewise, does `ksql.advertised.url` need the `confluent.controlcenter.` prefix?

**Comment by alex-cf** [comment]
Just an FYI that in 5.2, the properties `confluent.controlcenter.ksql.url` and `confluent.controlcenter.ksql.advertised.url` are being deprecated to accommodate configuring multiple ksql clusters. The new properties will be `confluent.controlcenter.ksql.<name>.url` and `confluent.controlcenter.ksql.<name>.advertised.url`

Specifics here: https://github.com/confluentinc/docs/blob/master/control-center/installation/configuration.rst#ksql-settings

**Comment by JimGalasyn** [comment]
Thank you @alex-cf, this is on my radar. I'll update the topic for 5.2 after I merge this puppy.

**Pull request merged**